packages/schema/src/types/schema.ts

```
@@ -0,0 +1,2230 @@
+
+    /**
+     * OptimizeCSSAssets plugin options.
+     *
+     * Defaults to true when `extractCSS` is enabled.
+     *
+     * @default false
+     *
+     * @see [css-minimizer-webpack-plugin documentation](https://github.com/webpack-contrib/css-minimizer-webpack-plugin).
+     */
+    // eslint-disable-next-line @typescript-eslint/no-empty-object-type
+    optimizeCSS: false | BasePluginOptions & DefinedDefaultMinimizerAndOptions<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)